test/suites/projects.sh

```diff
@@ -429,9 +429,16 @@ test_projects_images_default() {
   lxc project switch foo
   lxc project set foo "features.images" "false"
 
-  # The project can see images from the defaut project
+  # Create another project, without the features.images config.
+  lxc project create bar
+  lxc project set bar "features.images" "false"
+
+  # The project can see images from the default project
   lxc image list | grep -q testimage
 
+  # The image from the default project has correct profile assigned
+  ! lxc image show testimage | grep -qF "profiles: []" || false
```

Comment:
I think we should test explicitly for the presence of the default profile using something like:

```
fingerprint="$(lxc image list -c f --format json | jq -r .[0].fingerprint)"
lxc query /1.0/images/"${fingerprint}"?project=test1 | jq -r .profiles[0] | grep -Fx default 
```

Reply:
Done. Seems `-c f` has no effect together `--format json` by the way.